.github/workflows/rust.yml

```diff
@@ -54,6 +54,7 @@ jobs:
       - run: cargo test
       - run: cargo test --manifest-path fuzz/Cargo.toml
         if: ${{ matrix.rust }} == "stable"
+      - run: cargo test -p quinn-udp --benches
```

Comment:
See https://github.com/quinn-rs/quinn/pull/2010#issuecomment-2424749007 for reason for `-p quinn-udp`.